### PR TITLE
Announce Dune 3.8.X

### PR DESCRIPTION
**Pull request by tmattio**

### CODE REVIEW

**Comment by tmattio** [comment]
@emillon what do you think are features from this release that we should highlight?

**Comment by emillon** [comment]
I'd say `(concurrent)`, the fact that mdx that knows about `mld` files, and the fact that Dune knows about installed coq theories. Do you want some prose from us to describe that?

**Comment by tmattio** [comment]
@emillon I've added a couple of feature highlights and the announcements for Dune 3.8.1 and 3.8.2. I'm happy to merge if this looks good to you.

File: data/changelog/dune/2023-05-31-dune-concurrent-action.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Dune 3.8.0 introduced the new `concurrent` action. You can now use it instead of the `progn` action to execute actions concurrently.
+
+For instance:
+
+```
+(rule
+ (action
+  (concurrent
+   (with-outputs-to A (echo "I am file A.\n"))
```
Comment (emillon) [inline data/changelog/dune/2023-05-31-dune-concurrent-action.md]:
```suggestion
   (write-file A "I am file A.\n")
```

& similar below

**Pull request merged**